app/src/main/java/fr/gaulupeau/apps/Poche/ui/MainActivity.java

```diff
@@ -798,4 +799,24 @@ private void cancelConfigurationTest() {
         }
     }
 
+    private void showAddBagDialog() {
+        AlertDialog.Builder builder = new AlertDialog.Builder(this);
+        builder.setTitle(getString(R.string.nav_add));
+
+        final View view = getLayoutInflater().inflate(R.layout.dialog_add, null);
```

Comment:
We need to suppress the lint warning like [here](https://github.com/wallabag/android-app/blob/master/app/src/main/java/fr/gaulupeau/apps/Poche/ui/ReadArticleActivity.java#L437).